packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -310,7 +315,7 @@ class CheckboxListTile extends StatelessWidget {
   Widget build(BuildContext context) {
     final Widget control = Checkbox(
       value: value,
-      onChanged: onChanged,
+      onChanged: (enabled != null && !enabled!) ? null : onChanged,
```

Comment:
I suppose this could be: `enabled ?? false : null : onChanged`

Reply:
Thanks for the review, Probably you meant 

```dart
enabled ??  false ? null : onChanged
``` 
But, this wouldn't work for a case when `enabled` is `true`.
When enabled is true `onChanged` would be `null`.  For this API I think `onChanged` should be enabled (non null) only when `enabled` is `true`.

Reply:
We probably want to pass null when enabled is false. In this case, this should work: 
`(enabled ?? true) ? onChanged :null`

Reply:
NICE